root/static/css/form.css

```diff
@@ -1126,3 +1127,46 @@ form li.complex p.error{
 	font-weight:normal;
 	font-size:160%;
 }
+
+/*responsive*/
+@media all and (min-width: 50px) and (max-width: 400px){
+  form ul li.rightHalf {
+    width: 97% !important;
+  }
+}
+/*@media all and (min-width: 300px) and (max-width: 650px) {
+  table .responsive { 
+    margin: 5px 10px;
+    border: 1px solid #aaa;
+    border-width: 1px 1px 0 0;
+    border-spacing: 0;
+    width: 200px;
+  } 
+}
+@media all and (min-width: 90px) and (max-width: 299px) {
+  table .responsive { 
+    margin: 5px 10px;
+    border: 1px solid #aaa;
+    border-width: 1px 1px 0 0;
+    border-spacing: 0;
+    width: 100px;
+  } 
+}*/
+@media screen and (max-width: 640px) {
+	table {
+		overflow-x: auto;
+		display: block;
+	}
+}
+@media all and (min-width: 100px) and (max-width: 299px) {  
+  table {
+    overflow-x: auto;
+		display: block;
+	}
+}
+@media all and (min-width: 100px) and (max-width: 299px) {
+  form #username, #password {
+    width: 40px; 
+  }
+}
```

Comment:
In line 1132, the max-width 400px is set and here it's 299px. Why the difference?